### PR TITLE
Use formatted strings (f-strings)everywhere

### LINKED ISSUE #246: Use formatted strings everywhere
Python 3.6 onwards supports something called `f-strings`. Instead of `"x is " + str(x)`, you can write `f"x is {x}"`. f-strings use the same formatting syntax as the existing `format`, but are much cleaner.

I suggest using f-strings throughout our codebase, and in the future for new code.

### PR DESCRIPTION
**Pull request by michaelfu1029**
# Pull Request

## Description
Changing string literals to f-strings. Did not change team.py as it is currerently stubs, and webhook.py, as some functions are not merged yet(fixed in the other PR #247 )

## Ticket(s)

Closes #246

### CODE REVIEW

File: interface/slack.py [hunk 1]
```diff
@@ -13,42 +13,42 @@ def __init__(self, sc):
...
             "chat.postMessage",
             channel=channel_name,
             attachments=attachments,
             text=message
         )
         if 'ok' not in response:
-            logging.error("Message to channel {} failed with error: {}".
-                          format(channel_name, response['error']))
+            logging.error(f"Message to channel {channel_name} failed with "
+                          f"error: { response['error']}")
```
Comment (cheukyin699) [inline interface/slack.py]: Space between left bracket and `r` where there is no space between the right bracket: consistency

Comment (michaelfu1029) [reply interface/slack.py]: fixed

File: command/commands/user.py [hunk 2]
```diff
@@ -183,8 +183,8 @@ def edit_helper(self, user_id, param_list):
                 self.github.org_add_member(param_list["github"])
                 edited_user.github_username = param_list["github"]
             except GithubAPIException as e:
-                msg = "\nError adding user {} to GitHub organization".format(
-                    param_list['github'])
+                msg = (f"\nError adding user {param_list['github']} to GitHub"
```
Comment (rwblickhan) [inline command/commands/user.py]: I think the parentheses here can be removed.

File: interface/slack.py [hunk 3]
```diff
@@ -13,42 +13,42 @@ def __init__(self, sc, slack_channel=""):
...
+        logging.debug(f"Sending direct message to {slack_user_id}")
         response = self.sc.api_call(
             "chat.postMessage",
             channel=slack_user_id,
             text=message
         )
-        if 'ok' not in response or not response['ok']:
-            logging.error("Direct message to {} failed with error: {}".
-                          format(slack_user_id, response['error']))
+        if 'ok' not in response:
```
Comment (ravenCondol) [inline interface/slack.py]: why change ?

File: interface/slack.py [hunk 4]
```diff
@@ -13,42 +13,42 @@ def __init__(self, sc, slack_channel=""):
...
         response = self.sc.api_call(
             "chat.postMessage",
             channel=channel_name,
             attachments=attachments,
             text=message
         )
-        if 'ok' not in response or not response['ok']:
-            logging.error("Message to channel {} failed with error: {}".
-                          format(channel_name, response['error']))
+        if 'ok' not in response:
```
Comment (ravenCondol) [inline interface/slack.py]: here too, think the or is important here

**Pull request merged**